src/clique_command.erl

```diff
@@ -40,23 +40,23 @@ init() ->
     ok.
 
 %% @doc Register a cli command (i.e.: "riak-admin handoff status")
--spec register([string()], list(), list(), fun()) -> true.
+-spec register(['*' | string()], list(), list(), fun()) -> true.
```

Comment:
I'm inclined to think `register` should sanity check the command list to ensure that any wildcards are at the end.